# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ScriptKind.h
# Commit: a48aff5f2555
# Full Hash: a48aff5f2555af61a9bc1ee0d77f6058f47d0501
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 7e54a223001d -r a48aff5f2555 dom/script/ScriptKind.h
--- a/dom/script/ScriptKind.h	Thu Feb 17 15:57:51 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,18 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set ts=8 sts=2 et sw=2 tw=80: */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef mozilla_dom_ScriptKind_h
-#define mozilla_dom_ScriptKind_h
-
-namespace mozilla {
-namespace dom {
-
-enum class ScriptKind { eClassic, eModule, eEvent };
-
-}  // namespace dom
-}  // namespace mozilla
-
-#endif